anchor-reference-server/src/main/java/org/stellar/anchor/reference/service/CustomerService.java

```diff
@@ -222,6 +230,13 @@ public CustomerField createBankNumberField(String type) {
     field.setOptional(!type.equals(Customer.Type.SEP31_RECEIVER.toString()));
     return field;
   }
+  public CustomerField createClabeNumberField(String type) {
```

Comment:
nit: could you rename `type` to `customerType`? I had to look into the code to figure out what type means, as I don't find it self-explanatory in the context of this method.

Reply:
Sure, I've updated it.